### PR TITLE
{lang}[dummy/] Julia v0.6.0

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/j/Julia/Julia-0.6.0-linux-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (verdurin) [inline easybuild/easyconfigs/j/Julia/Julia-0.6.0-linux-x86_64.eb]: @pescobar please fix the URL to use the new repo

File: easybuild/easyconfigs/j/Julia/Julia-0.6.0-linux-x86_64.eb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+version = '0.6.0'
+versionsuffix = '-linux-x86_64'
+
+homepage = 'https://julialang.org'
+description = """Julia is a high-level, high-performance dynamic programming language for numerical computing"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://julialang-s3.julialang.org/bin/linux/x64/%(version_major_minor)s/']
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
```
Comment (verdurin) [inline easybuild/easyconfigs/j/Julia/Julia-0.6.0-linux-x86_64.eb]: Please add an SHA256 checksum here.

**Comment by boegel** [comment]
@pescobar Ping on remarks?

**Comment by pescobar** [comment]
@boegel I undertand that you prefer build from sources and also this is an old version

it's fine to close the PR. It can stay in the list of old PRs as reference for others

**Pull request closed without merging**